pkgs/top-level/python-packages.nix

```diff
@@ -25797,4 +25797,21 @@ in modules // {
     };
   };
 
+  topydo = buildPythonPackage rec {
+    name = "topydo-${version}";
+    version = "0.9";
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
The pypi page only proposes to download a wheel package. Is there a way to use this as a source (or a hidden way to download a tarball from pypi) ?